metrics/storage/webtooling.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2021 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'web tooling benchmark'
+# https://github.com/v8/web-tooling-benchmark
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+TEST_NAME="web-tooling"
```

Comment:
this var is not being used below

Reply:
@chavafg is being use here https://github.com/kata-containers/tests/blob/620f5f280a749703229d051bf33a286058dd2743/metrics/lib/common.bash#L148